UPDATE_CHECKLIST.md

```diff
@@ -6,6 +6,7 @@ This checklist is seen as a guide to update the existing deployment.
 
 - [ ] Make sure you are using Node v16
 - [ ] Verify that all changes of this update are reflected in [contracts/CHANGELOG.md](packages/contracts/CHANGELOG.md) by comparing the diff with the previous release commit.
+- [ ] Check that tests checking for storage corruption using OZ's `hardhat-upgrades` package exists for every upgradeable contract and that test the upgradew from all prior versions to the current version
```

Comment:
"upgradew" has a typo.

should we provide a link to `test-utils/upps-upgradeable.ts` and be explicit that those check must have been used?

Reply:
Fixed in c3181e902ff92fcd09f2966eb1fa282e4558b1f0